api/internal/git/repospec.go

```diff
@@ -212,18 +222,24 @@ func peelQuery(arg string) (string, string, time.Duration, bool) {
 	return parsed.Path, ref, duration, submodules
 }
 
-func parseHostSpec(n string) (string, string) {
+func parseHostSpec(n string) (string, string, error) {
```

Comment:
Maybe I miss something, @mightyguava Do you plan to return some actual error? It looks like only ln 255 will return "nil".

Reply:
Line 271 can return non-nil https://github.com/kubernetes-sigs/kustomize/blob/1841f0b94fc414115d88b3957ae01dd405a0c4ee/api/internal/git/repospec.go#L271

Reply:
ack